.travis.yml

```diff
@@ -41,20 +51,37 @@ matrix:
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.7
+        - PLAT=i686
         - NP_BUILD_DEP="1.14.5"
         - NP_TEST_DEP="1.14.5"
+
     - os: linux
+      arch: arm64-graviton2
       env:
-        - MB_PYTHON_VERSION=3.8
+        - MB_PYTHON_VERSION=3.6
+        - NP_BUILD_DEP="1.11.3"
+        - NP_TEST_DEP="1.11.3"
+        - MB_ML_VER=2014
+        - PLAT=aarch64
+        - DOCKER_TEST_IMAGE=multibuild/xenial_arm64v8
+    - os: linux
+      arch: arm64-graviton2
```

Comment:
Does the build pass if you use the regular travis arm64 machines?

Reply:
Unfortunately, no: https://travis-ci.org/github/MacPython/h5py-wheels/builds/735063598
```
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received
The build has been terminated
```

Reply:
@mattip You could try changing the no-output timeout and re-running it.